### PR TITLE
Update generated files

### PR DESCRIPTION
**Pull request by jon-a-nygaard**
Not sure why these changes appear in the master, but would be nice to have them commited to avoid accidentally including them in a irrelevant commit.

### CODE REVIEW

File: errors/errors.json [hunk 1]
```diff
@@ -1,118 +1,118 @@
...
+		"text": "<h1>Can't plot zero or subzero values on a logarithmic axis</h1><p>This error occurs in the following situations:</p><ul><li>If a zero or subzero data value is added to a logarithmic axis</p><p>- If the minimum of a logarithimic axis is set to 0 or less</p><p>- If the threshold is set to 0 or less</p><p>Note: As of Highcharts 5.0.8 it's possible to bypass this error message by</p><p>setting <code>Axis.prototype.allowNegativeLog</code> to true, and add custom conversion</p><p>functions.</p><p><a href=\"https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/samples/highcharts/yaxis/type-log-negative/\">View live demo</a>.</li></ul>",
```
Comment (bre1470) [inline errors/errors.json]: List markdown does not work properly.

File: js/error-messages.js [hunk 2]
```diff
@@ -16,118 +16,118 @@ import H from './parts/Globals.js';
...
-            "errors/23/readme.md",
-            "errors/24/readme.md",
-            "errors/25/readme.md",
-            "errors/26/readme.md",
-            "errors/27/readme.md",
-            "errors/28/readme.md",
-            "errors/29/readme.md",
-            "errors/30/readme.md",
-            "errors/31/readme.md"
+            "errors\\10\\readme.md",
```
Comment (bre1470) [inline js/error-messages.js]: File pattern changed style.

File: errors/errors.json [hunk 3]
```diff
@@ -1,118 +1,118 @@
...
-			"errors/23/readme.md",
-			"errors/24/readme.md",
-			"errors/25/readme.md",
-			"errors/26/readme.md",
-			"errors/27/readme.md",
-			"errors/28/readme.md",
-			"errors/29/readme.md",
-			"errors/30/readme.md",
-			"errors/31/readme.md"
+			"errors\\10\\readme.md",
```
Comment (bre1470) [inline errors/errors.json]: Windows style, but POSIX preferred - probably needs a change in tools/errors.js

File: errors/errors.json [hunk 4]
```diff
@@ -1,118 +1,118 @@
...
 	},
 	"11": {
 		"title": "Can't link axes of different type",
-		"text": "<h1>Can't link axes of different type</h1><p>This error occurs when using the <code>linkedTo</code> option to link two axes of different types, for example a logarithmic axis to a linear axis. Highcharts can't link these because the calculation of ticks, extremes, padding etc. is different.</p>"
+		"text": "<h1>Can't link axes of different type</h1><p>This error occurs when using the <code>linkedTo</code> option to link two axes of</p><p>different types, for example a logarithmic axis to a linear axis. Highcharts</p><p>can't link these because the calculation of ticks, extremes, padding etc. is</p><p>different.</p>"
```
Comment (bre1470) [inline errors/errors.json]: Line breaks result in new paragraph

**Comment by jon-a-nygaard** [comment]
Closing this PR as it turned out the issue was due to the Windows compatibility of the generator, and not due to outdated state of the committed files.

**Pull request closed without merging**